policyengine_canada/parameters/gov/provinces/yt/tax/income/credits/basic_personal_amount/base_amount.yaml

```diff
@@ -0,0 +1,9 @@
+description: Yukon provides this base amount under the basic personal amount.
+values:
+  2023-01-01: 13_521
+metadata:
+  unit: currency-CAD
+  label: Yukon basic personal amount base amount
+  reference:
+  - title: Worksheet for the Yukon 2023 Personal Tax Credits Return
+    href: https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt-ws/td1yt-ws-23e.pdf
```

Comment:
add [this](https://www.canada.ca/content/dam/cra-arc/formspubs/pbg/td1yt/td1yt-23e.pdf) source